config/default/field.field.paragraph.stanford_gallery.field_photoalbum_style.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 8bfc5e7f-2ac8-497c-a12d-2e86d41257d1
```

Comment:
FWIW I've been preferring to use paragraph behaviors for these type of things. It's nice to be able to separate the styles with the actual content.

Reply:
@pookmish Seems like a sensible place to organize that type of functionality. In this release we were tasked with updating the labels and sort of this "field" and noted that this paragraph in particular was an outlier compared to the others. We opted to change this one paragraph instead of converting the rest to a behaviors as it seemed like using a configurable field was the preferred convention given its prevalence in the existing architecture. 
If you strongly prefer the behavior implementation we can create a tech-debt ticket to revert this functionality back to a behavior and convert the following components in the same manner:
- callout box
- color band
- postcard
- spotlight